framework-plugins/lisk-framework-report-misbehavior-plugin/src/report_misbehavior_plugin.ts

```diff
@@ -91,24 +99,61 @@ export class ReportMisbehaviorPlugin extends BasePlugin {
 
 	// eslint-disable-next-line class-methods-use-this
 	public get actions(): ActionsDefinition {
-		return {};
+		return {
+			// eslint-disable-next-line @typescript-eslint/require-await
+			authorize: async (action: ActionInfoObject): Promise<{ result: string }> => {
```

Comment:
we can remove async here

Reply:
I think the action interface requires to be async

Reply:
oh then I have to also make one of the actions as async in monitor_plugin https://github.com/LiskHQ/lisk-sdk/blob/efda98cfddba88841f21cf4e78959f26fcb7d85d/framework-plugins/lisk-framework-monitor-plugin/src/monitor_plugin.ts#L84